books_handler.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'book'
+
+class BooksHandler
+  attr_reader :books
+
+  def initialize
+    @books = []
+  end
+
+  def add_book
+    print 'Title: '
+    title = gets.chomp
+    print 'Author: '
+    author = gets.chomp
+    @books << Book.new(title, author)
+    puts 'Book created successfully'
+  end
```

Comment:
- [ ] @Team Great work adding a method to add new book, however, to achieve the Single-responsibility Principle all requests for input and input validation should've been placed in the main.rb file and/or classes/methods that are responsible for requesting or validating input. You can add a new file whose main role is to ensure the user can add new persons.

Reply:
Solve